source/Patches/MeetingIntroPatch.cs

```diff
@@ -0,0 +1,40 @@
+﻿using HarmonyLib;
+using System.Collections.Generic;
+using UnhollowerBaseLib;
+
+using PlayerData = GameData.PlayerInfo;
+
+namespace TownOfUs
+{
+    
+    public static class MeetingIntroPatch
+    {
+        public static List<PlayerData> DeadPlayers =
+            new List<PlayerData>();
```

Comment:
`new List<PlayerData>(Il2CppEqualityComparer<PlayerData>.Instance);`

Reply:
Should actually be `HashSet` as well, but again it still requires the `Il2CppEqualityComparer` from Reactor to be added to the code base here.